devTools/backupWordpressImages/backup-wordpress-images.ts

```diff
@@ -0,0 +1,142 @@
+import { createReadStream } from "fs"
+import { google } from "googleapis"
+import fs from "fs/promises"
+import path from "path"
+
+import { Gdoc } from "../../db/model/Gdoc/Gdoc.js"
+import { GDOCS_IMAGES_BACKPORTING_TARGET_FOLDER } from "../../settings/serverSettings.js"
+
+const WORDPRESS_UPLOADS_PATH = path.join(
+    process.cwd(),
+    "wordpress/web/app/uploads/2008"
```

Comment:
This should just be `wordpress/web/app/uploads/`

Scoping it to 2008 for testing until we're ready as it only has 6 files in it.